product-backend/src/test/java/validator/RegisterValidatorTest.java

```diff
@@ -0,0 +1,107 @@
+package validator;
+
+import org.junit.jupiter.api.Test;
+import org.kainos.ea.cli.RequestEmployee;
+import org.kainos.ea.validator.RegisterValidator;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RegisterValidatorTest {
+
+    RegisterValidator registerValidator = new RegisterValidator();
+
+    @Test
+    public void isValidEmployee_shouldReturnTrue_whenValidDataProvided() throws Exception {
+        RequestEmployee employee = new RequestEmployee(
+                "test@kainos.com",
+                "Test1234!",
+                "Admin"
+        );
+
+        assertTrue(registerValidator.validate(employee));
+    }
+
+    @Test
+    public void isValidEmployee_shouldThrowException_whenPasswordHasNoUppercase() {
+        RequestEmployee employee = new RequestEmployee(
+                "test@kainos.com",
+                "test1234!",
+                "Admin"
+        );
+
+        assertThrows(Exception.class,
```

Comment:
With just catching Exception, and not an exact Exception, or checking for the message you risk a potential false positive result <- that happens when the method will throw exception but because of different reason then the one being tested.